src/color/grayscale.js

```diff
@@ -26,8 +26,9 @@ import toColorString from './toColorString'
  * }
  */
 function grayscale(color: string): string {
+  const hslColor = parseToHsl(color)
   return toColorString({
-    ...parseToHsl(color),
```

Comment:
@bhough one nitpick … can you revert that change? No need to assign the hslColor here